docs/pages/jpscore/jpscore_introduction.md

```diff
@@ -4,12 +4,12 @@ keywords: simulation
 tags: [jpscore, getting_started]
 sidebar: jupedsim_sidebar
 permalink: jpscore_introduction.html
-summary: jpscore is the simulation module of JuPedSim. It's a command line tool to simulate the evacuate of pedestrians in continous space.
+summary: jpscore is the simulation module of JuPedSim. It's a command line tool to simulate the evacuate of pedestrians in continuous space.
```

Comment:
Why are these typo corrections included in a PR related to sources? Please move to a seperate PR

Reply:
removed from this PR